contracts/escrow721/src/state.rs

```diff
@@ -0,0 +1,3 @@
+use cw_storage_plus::Map;
+
+pub const CLASS_STORAGE: Map<String, String> = Map::new("class_storage");
```

Comment:
It's preferable to use `&str` here than owned strings. Did you run into issues using `&str`?

Reply:
Can you add a comment about what these strings represent?

Reply:
Changed to <&str, String> with a comment